ui/src/app/components/apps-panel.spec.tsx

```diff
@@ -136,7 +143,7 @@ describe('AppsPanel', () => {
   );
 
   // these tests assume that there are no Jupyter runtimes
-  // so what these tests actually show is whether Cromwell is an ActiveApp
```

Comment:
nit: the original text remains true here.  But this text is also true.  I dunno.

Reply:
Yeah, there's no great way to generalize a test for "all apps" since you need to choose one of them for the test. I think the test is reflective enough of all apps at the moment that I'm ok with it.

One alternative is to re-run these tests for each app type but that seems more complex than it's worth.